packages/design-system/src/styles/settings/variables/_color.scss

```diff
@@ -84,6 +84,11 @@ $color-base-inverse: $color-white !default;
 $color-muted-inverse: #bac5cf !default;
 
 // Focus
+// Default focus styles
+$color-focus: #3e94cf !default;
```

Comment:
I think we should still probably remove these

Reply:
I was thinking we should keep them because they are used when the gold focus styles are disabled.

Reply:
Yeah you are right, and this actually made me realize something, in the few places that use `$color-focus`, we dont use the default browser focus styles. Was this how it was originally, or did previously everywhere used `$color-focus` and our own custom focus shadow? @davidakennedy do you happen to remember?

Reply:
A comparison: 
<img width="244" alt="Screen Shot 2020-06-25 at 2 25 44 PM" src="https://user-images.githubusercontent.com/5424713/85786315-c2cee780-b6ef-11ea-9581-d57bc5324c1d.png">
<img width="311" alt="Screen Shot 2020-06-25 at 2 25 38 PM" src="https://user-images.githubusercontent.com/5424713/85786333-c7939b80-b6ef-11ea-98d4-d561c0f8bff7.png">

Reply:
I tried to revert it back to how it was previously - which involved going through DKs PR https://github.com/CMSgov/design-system/pull/510/files

Reply:
Yeah, @line47 is right, @bernardwang. That was one of the biggest changes in my original PR, it applied things everywhere vs. selectively.